utils/variables/_layers.scss

```diff
@@ -0,0 +1,10 @@
+// @todo where comes this values from, do and why does they make sense?
```

Comment:
Values come from here: https://www.sitepoint.com/better-solution-managing-z-index-sass/
Some cases you'd need -ve values especially with opacity (However we don't have such a case yet)